Makefile

```diff
@@ -1,25 +1,58 @@
 #!/usr/bin/make -f
 
-REGISTRY=skpr/nginx
+REGISTRY_BASE=skpr/nginx
+REGISTRY_PHP_FPM=skpr/nginx-php-fpm
+REGISTRY_DRUPAL=skpr/nginx-drupal
+
 VERSION_TAG=v2-latest
 ARCH=amd64
 
 build:
-	docker build -t ${REGISTRY}:${VERSION_TAG}-${ARCH} .
-	docker run -it --rm --hostname=php-fpm --read-only ${REGISTRY}:${VERSION_TAG}-${ARCH} nginx -t
-	docker build --build-arg VERSION_TAG=${VERSION_TAG} --build-arg ARCH=${ARCH} -t ${REGISTRY}:dev-${VERSION_TAG}-${ARCH} dev
-	docker run -it --rm --hostname=php-fpm --read-only ${REGISTRY}:dev-${VERSION_TAG}-${ARCH} nginx -t
+	# Building Base
+	docker build -t ${REGISTRY_BASE}:${VERSION_TAG}-${ARCH} base
```

Comment:
we can do `$(eval IMAGE=${REGISTRY_BASE}:${VERSION_TAG}-${ARCH}` to remove a bunch of duplicate strings